docs/blog/2025-05-12-noir-integration.md

```diff
@@ -106,9 +106,9 @@ Our current implementation draws heavily from the zkPassport team’s work, whic
 | Platforms | Current support target        | Support |
 | --------- | ----------------------------- | ------- |
 | iOS       | `aarch64-apple-ios`           | ✅      |
-| iOS       | `aarch64-apple-ios-sim`       | ❌      |
```

Comment:
I only want to update the 3 platform support. Others were done by auto-formatting.